lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -131,6 +131,20 @@ def test_basic(self):
             }
         )
 
+
+@attr('shard_2')
+@ddt.ddt
+@mock.patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
+class GetCourseTestBlackouts(UrlResetMixin, ModuleStoreTestCase):
+    @mock.patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
+    def setUp(self):
+        super(GetCourseTestBlackouts, self).setUp()
+        self.course = CourseFactory.create(org="x", course="y", run="z")
```

Comment:
question: how is this isolated to only this test? (I'm guessing it is; just curious where.)